samples/dotnet-svcutil.xmlserializer-instructions.md

```diff
@@ -0,0 +1,78 @@
+
+You can start using the tool today following the instructions below. 
+
+## Prerequisites
+
+The following is required for svcutil.xmlserializer to work. 
+
+* [.NET Core SDK 2.1.2 or later](https://www.microsoft.com/net/download/windows)
+* [.NET Core runtime 2.1.0-preview1 or later](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.1.0-preview1-download.md)
+
+You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
+
+## Instructions
+
+Here are the step by step instructions on how to use dotnet-svcutil.xmlserializer in a .NET Core console application.
+
+1. Create a WCF Service named 'MyWCFService' using default template 'WCF Service Application' in .NET Framework.  Add [XmlSerializerFormat] attribute on the service method like the following
+    ```c#
+    [ServiceContract]
+    public interface IService1
+    {
+        [XmlSerializerFormat]
+        [OperationContract(Action = "http://tempuri.org/IService1/GetData", ReplyAction = "http://tempuri.org/IService1/GetDataResponse")]
+        string GetData(int value);
+    }
+    ```
+2. Create a .NET Core console application as WCF client application that targets at netcoreapp 2.1, e.g. create an app named 'MyWCFClient' with the command,
+    ```
+    dotnet new console --name MyWCFClient
+    ```
+    And make sure your csproj targets at netcoreapp 2.1 as the following in .csproj
+    ```xml
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    ```
+3. Add package reference for servicemodel
+   
+   Run command: `dotnet add packages System.ServiceModel.Http -v 4.4.2`
+
+4. Add WCF Client code
+    ```c#
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var myBinding = new BasicHttpBinding();
+            var myEndpoint = new EndpointAddress("http://localhost:2561/Service1.svc"); //Fill your service url here
+            var myChannelFactory = new ChannelFactory<IService1>(myBinding, myEndpoint);
+            IService1 client = myChannelFactory.CreateChannel();
+            string s = client.GetData(1);
+            ((ICommunicationObject)client).Close();
+        }
+    }
+
+    [ServiceContract]
+    public interface IService1
+    {
+        [XmlSerializerFormat]
+        [OperationContract(Action = "http://tempuri.org/IService1/GetData", ReplyAction = "http://tempuri.org/IService1/GetDataResponse")]
+        string GetData(int value);
+    }
+    ```
+5. Edit the .csproj and add a reference to the dotnet-svcutil.xmlserializer package. For example,
```

Comment:
You should know the filename of the csproj file based on the dotnet command ran earlier